### DISCUSSION TITLE
Template

### DISCUSSION THREAD

**Discussion by eduardoagr** [CommunityToolkit/Maui#1505] [New Feature Discussions]
Many of us, that use MVVM, we have to waste time, installing MCT an MIcrosoft MVVM padckage as well as making folders for Model,View,ViewModel. Why nor make it posible, that the intalation of MCT, comes with Microsoft MVVM Package and create the foder structure

**Comment by bijington** [comment]
I advocate for using Matt Lacey's extension MauiAppAccelerator

https://github.com/mrlacey/MauiAppAccelerator

**Comment by TheCodeTraveler** [comment]
I recommend using the MAUI App Accelerator plugin. 

I use it for my templates. It allows you to include all of the Toolkits in File -> New Project. 

https://marketplace.visualstudio.com/items?itemName=MattLaceyLtd.MauiAppAccelerator

**Reply by eduardoagr** [reply]
thanks guys
